r/tools/nixlibs.R

```diff
@@ -402,10 +407,11 @@ with_s3_support <- function(env_vars) {
       cat("**** S3 support not available for gcc < 4.9; building with ARROW_S3=OFF\n")
       arrow_s3 <- FALSE
     } else if (!cmake_find_package("CURL", NULL, env_vars)) {
+      # curl on macos should be installed, so no need to alter this for macos
       cat("**** S3 support requires libcurl-devel (rpm) or libcurl4-openssl-dev (deb); building with ARROW_S3=OFF\n")
       arrow_s3 <- FALSE
     } else if (!cmake_find_package("OpenSSL", "1.0.2", env_vars)) {
-      cat("**** S3 support requires openssl-devel (rpm) or libssl-dev (deb), version >= 1.0.2; building with ARROW_S3=OFF\n")
+      cat("**** S3 support requires openssl-devel (rpm), libssl-dev (deb), openssl (brew), version >= 1.0.2; building with ARROW_S3=OFF\n")
```

Comment:
This message could use a rewrite for clarity

Reply:
What aspect of it do you think needs more clarity? I'm happy to polish it + the ones above it, but I'm not sure what direction is lacking.

Reply:
Ah, wonderful thanks!